v3/plugins/redhat/dependency-analytics/0.1.0/meta.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v2
+publisher: redhat
+name: dependency-analytics
+version: 0.1.0
+type: VS Code extension
+displayName: Dependency Analytics
+title: Insights about your application dependencies
+description: Insights about your application dependencies, Security, License compatibility and AI based guidance to choose appropriate dependencies for your application.
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+repository: https://github.com/fabric8-analytics/fabric8-analytics-vscode-extension
+category: Other
+firstPublicationDate: '2020-07-14'
+spec:
+  containers:
+    - image: "quay.io/eclipse/che-sidecar-dependency-analytics:0.0.13-a38cb0c"
```

Comment:
Are you okay with using this container sidecar? The new version of the plugin doesn't need any new dependencies?

Reply:
Yeah, it is working fine with the existing container sidecar. The new major updates are mostly for vulnerability data improvement by integration with Snyk and others are on back-end API side.